barterdude/hooks/metrics/prometheus/__init__.py

```diff
@@ -27,14 +26,13 @@ def __init__(
         barterdude: BarterDude,
         labels: dict,
         path: str = "/metrics",
-        registry: CollectorRegistry = REGISTRY
-
+        registry: CollectorRegistry = None
     ):
-        self.__registry = registry
+        self.__registry = registry or CollectorRegistry()
```

Comment:
Can't we use default registry as fallback?

Reply:
That was the bug, we were using the default registry as the fallback, which resulted in trying to register the same metrics more than once. Using a new registry avoids this problem.